src/sources/http_server.rs

```diff
@@ -133,6 +133,12 @@ pub struct SimpleHttpConfig {
     #[configurable(metadata(docs::examples = "vector_http_path"))]
     path_key: OptionalValuePath,
 
+
+    /// The event key in which the remote ip is stored.
```

Comment:
Since we are using the log_schema as the default for this one, I think this comment should follow what we do in other components that do this. There are a few but this is one example: https://github.com/vectordotdev/vector/blob/56486bafe6ce41a7c92a11ccd0e2cf6e8f7ef838/src/sources/docker_logs/mod.rs#L68-L72 (ofc, mentioning the remote IP though)